pandas/tests/io/json/test_pandas.py

```diff
@@ -985,12 +985,46 @@ def test_tz_range_is_utc(self):
         df = DataFrame({'DT': dti})
         assert dumps(df, iso_dates=True) == dfexp
 
-    def test_read_jsonl(self):
+    def test_read_inline_jsonl(self):
         # GH9180
         result = read_json('{"a": 1, "b": 2}\n{"b":2, "a" :1}\n', lines=True)
         expected = DataFrame([[1, 2], [1, 2]], columns=['a', 'b'])
         assert_frame_equal(result, expected)
 
+    @pytest.yield_fixture(scope="function")
+    def testbucket_conn(self):
```

Comment:
I haven't looked, but can you use the fixture defined in `pandas.tests.io.parser.test_network`? I think if you use `s3_resource`. You may still nee the `s3_resource.create_bucket`, but you shouldn't have to worry about moto.

Reply:
@TomAugspurger I was working on trying to reuse the code here. In order to reuse across moduled I'd probably need to move the `s3_resource` and its dependent fixtures to `conftest.py` so they'd be available to my test modules.

This may also entail changing how pointing to the data files that are mock-uploaded works since that code would no longer live in `test_network`. Do you agree with this? Or should I keep my implementation?